app/models/user.rb

```diff
@@ -384,6 +384,12 @@ def set_roles(role_list)
 
   public
 
+  # called to mark a user as "Active"
+  def update_active
+    REDIS_GENERAL.sadd("last_login_list", login)
+    REDIS_GENERAL.set("last_active_#{login}", Time.now)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.